setup/teardown.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Teardown
+description: Learn how to teardown Coder and its associated infrastructure.
+---
+
+This guide shows you how to teardown Coder and the cluster it is deployed on.
+
+**Note:** these instructions help you remove infrastructure spun up via our
+[Kubernetes Documentation](https://coder.com/docs/setup/kubernetes), and do not
+include teardown steps for any additional cluster resources that may have been deployed.
+
+## Amazon Elastic Kubernetes Service (EKS)
+
+1. Make sure you're running `eksctl` version 0.37.0 or later by running
+the following command:
+
+```bash
+eksctl version
+```
+
+2. List all services in the cluster by running the following command:
+
+```bash
+kubectl get svc --all-namespaces
+```
+
+3. Delete any services that have an `EXTERNAL-IP` value
+by running the following command:
+
+```bash
+kubectl delete svc <service-name>
+```
+
+4. Delete the cluster and its underlying nodes by
+running the following command:
+
+```bash
+eksctl delete cluster --name <prod>
+```
+
+## Azure Kubernetes Serivce (AKS)
+
+1. Ensure the environment variable for `RESOURCE_GROUP` is set
+to the one you want to delete in Azure by running the following command:
+
+```bash
+echo $RESOURCE_GROUP
+```
+
+2. Delete the cluster by running the following command:
+
+```bash
+az group delete --resource-group $RESOURCE_GROUP
+```
+
+## Google Kubernetes Engine (GKE)
+
+1. Ensure the environment variables for `PROJECT_ID` and `CLUSTER_NAME` are still
+set to the ones you want to delete in Google Cloud Platform (GCP).
+Do this by running the following commands:
+
+```bash
+echo $PROJECT_ID
+echo $CLUSTER_NAME
+```
```

Comment:
Similar to the Azure comment, may be worth also providing the commands to set the values or changing this to "set the values" instead of "check the values"

Reply:
Noted - will add this in.